src/acc.rs

```diff
@@ -305,6 +312,11 @@ impl Sexp for Paintable {
     }
 
     fn to_sexp_with_config(&self, config: &SexpConfig) -> String {
+        fn map_to_sexp<'a>(kv: impl Iterator<Item = (&'a String, &'a String)>) -> String {
```

Comment:
Ok, that's way nicer. Question: Is that always what we want for Map<K, V>? If so, you could implement `ToSExp` for `Map<K, V> where V: ToSExp` and `Map<K, String>`. Likely you should also require `K: ToSExp` or `K: Format`